src/test/scala/org/neo4j/spark/service/Neo4jQueryServiceTest.scala

```diff
@@ -188,7 +334,7 @@ class Neo4jQueryServiceTest {
       "WHERE ((source.name = 'John Doe' OR target.name = 'John Doraemon' OR source.name = 'Jane Doe') " +
       "AND (target.age = 34 OR target.age = 18) " +
       "AND rel.score = 12) " +
-      "RETURN source, rel, target", query)
+      "RETURN rel, source AS source, target AS target", query)
```

Comment:
why we have this?

Reply:
it would require additional logic to set or not set an alias if the names are equal. I can add it but I don't think it's worth it since this is invisible to the user and / or causes overhead on the query performance